src/main/java/com/aws/greengrass/componentmanager/plugins/docker/exceptions/DockerImageDeleteException.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.aws.greengrass.componentmanager.plugins.docker.exceptions;
+
+import static com.aws.greengrass.deployment.errorcode.DeploymentErrorCode.DOCKER_RMI_ERROR;
+
+public class DockerImageDeleteException extends DockerImageDownloadException {
```

Comment:
I don't think that this should be a subclass of the download exception. A deletion exception is not related to downloading.